docs/README.md

```diff
@@ -9,7 +9,6 @@
 
 * [Working with Projects](usage/working-with-projects.md)
 * [Project Operations](usage/project-operations.md)
-* [Working with Service Accounts](usage/working-with-service-accounts.md)
```

Comment:
right, but instead we should reference usage/gardener-api.md (Accessing the Gardener API)